Doc/library/doctest.rst

```diff
@@ -277,13 +277,34 @@ Which Docstrings Are Examined?
 The module docstring, and all function, class and method docstrings are
 searched.  Objects imported into the module are not searched.
 
-In addition, if ``M.__test__`` exists and "is true", it must be a dict, and each
+In addition, there are cases when you want tests to be part of a module but not part
+of the help text, which requires that the tests not be included in the docstring.
+Doctest looks for a module-level variable called ``__test__`` and uses it to locate other
+tests. If ``M.__test__`` exists and "is true", it must be a dict, and each
```

Comment:
Actually, we could just use the word truthy, we've used it elsewhere.

```suggestion
tests. If ``M.__test__`` exists and is truthy, it must be a dict, and each
```

Reply:
I tend to say "evaluates true" but truthy is concise, and used other places (I believe js uses the term liberally)

Reply:
@Unique-Usman Please use either `is truthy` or `evaluates true` instead of `"is true"`.